sim/druid/burning_treant.go

```diff
@@ -16,10 +16,16 @@ type BurningTreant struct {
 }
 
 func (druid *Druid) NewBurningTreant() *BurningTreant {
-	var burningTreantBaseStats = stats.Stats{stats.SpellCritPercent: 5}
+	baseStats := stats.Stats{stats.SpellCritPercent: 0}
```

Comment:
Ah yea, I guess it's the same findings I had with Fiery Imp from Warlock 2P

Reply:
Out of all the data we gathered, there isn't a single log where fireseed has crit.

Reply:
Was this tested without the 5% raid debuff?

Reply:
Was just double checking. both Warlock and Mage show 0 base crit and 0 inherited crit 👍